src/lib/format-locale-number.ts

```diff
@@ -0,0 +1,11 @@
+import { BigNumber } from "./bignumber";
+
+export const formatLocaleFixedNumber = (value: BigNumber, decimals?: number) => {
```

Comment:
Why exactly do we need this function in place of just using `formatNumber` in `src/lib/format-number.ts`

Reply:
Will do as https://github.com/vegaprotocol/token-frontend/issues/971